docs/robotics/perception/vision.md

```diff
@@ -1,23 +1,18 @@
-# Vision
-
-TODO: Mention all nodes or just the important ones?
-Nodes of questionable importance:
-
--   Field color detection
--   Camera matrix provider
-
-TODO: Add images
+Vision is the by far most important aspect of the robot's perception.
+The NAO has two cameras, one on the top and one on the bottom of the head, which both can operate at a resolution of 640x480 at 30 fps or at 2560x1080 at 1fps.
+For more details, have a look at the [documentation](http://doc.aldebaran.com/2-8/family/nao_technical/video_naov6.html) by Aldebaran.
 
 The vision cycler runs twice in two separate threads to process the images from the top and bottom camera in parallel.
 
-Image resolution is determined by the hardware interface, but is currently set to 640x480 pixels for performance reasons.
-Most of the vision pipeline happens on a segmented version of the image for the same reason.
+Image resolution is determined by the hardware interface and is currently set to 640x480 pixels, since the higher refresh rate is more important than the higher resolution and to decrease the reuquired processing power.
```

Comment:
```suggestion
Image resolution is determined by the hardware interface and is currently set to 640x480 pixels, since the higher refresh rate is more important than the higher resolution and to decrease the required processing power.
```